hnginternship5/apoe4_mobile
# Post daily questions

### Title of PR
Implemented Daily questions answers post to the server

### Feature Implemented
Submit Daily question using Submit Button

### How to test PR
When the user gets to the Today page, the user should see daily questions and also be able to submit the answers.
